tests/app/dao/test_templates_dao.py

```diff
@@ -49,13 +48,14 @@ def test_create_only_one_template(
     if template_type == EMAIL_TYPE:
         data.update({'subject': 'subject'})
     template = Template(**data)
-    dao_create_template(template)
-
-    persisted_template = notify_db_session.session.get(Template, template.id)
-    assert persisted_template == template
+    try:
+        dao_create_template(template)
 
-    # Teardown
-    template_cleanup(notify_db_session.session, template.id)
+        persisted_template = notify_db_session.session.get(Template, template.id)
+        assert persisted_template == template
+    finally:
```

Comment:
If we have to manually teardown it needs to be in a finally so it tears down even if the test fails.

Reply:
Only the "assert" needs to be in the above "try".  There's nothing to tear down if "dao_create_template" fails.